go/vt/sqlparser/ast_funcs.go

```diff
@@ -872,6 +853,11 @@ func (node *Union) SetLock(lock Lock) {
 	node.Lock = lock
 }
 
+// SetInto sets the into clause
+func (node *Union) SetInto(into *SelectInto) {
```

Comment:
this does not look correct. there is no INTO on the UNION clause, is there?

Reply:
```sql
MySQL root@(none):test> (select 1) union (select 2) limit 1 into @myvar
Query OK, 1 row affected
Time: 0.001s
```
MySQL allows into in a union clause too

Reply:
TIL there is INTO on UNION